### PR TITLE
Edited Functions Chapter Excercise 14

### PR DESCRIPTION
**Pull request by hangde01**

### CODE REVIEW

**Comment by bnmnetp** [comment]
I think you accidentally added the build_info file too.

**Comment by hangde01** [comment]
I removed the build_info file

File: _sources/Strings/Exercises.rst [hunk 1]
```diff
@@ -174,24 +177,24 @@ Exercises
...
+
               def mirror(mystr):
                   # your code here
-        
+
               testEqual(mirror('good'), 'gooddoog')
               testEqual(mirror('Python'), 'PythonnohtyP')
               testEqual(mirror(''), '')
               testEqual(mirror('a'), 'aa')
-
```
Comment (bnmnetp) [inline _sources/Strings/Exercises.rst]: @hangde01 These should be replaced with the new style unittests....

**Comment by hangde01** [comment]
I combined the changes that Kyle and I made so that all the String chapter changes are in one file, and the old style testEqual() functions have been replaced.

**Pull request merged**